lib/middleman-sprockets/extension.rb

```diff
@@ -75,11 +75,7 @@ def processible? r
 
       Contract String => Bool
       def check_asset path
-        if environment[path]
-          true
-        else
-          false
-        end
+        path.present? && environment[path]
```

Comment:
It looks like the return value of this will no longer be a boolean so it'll break the contract -- this should be cast to true or false